client/packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -137,22 +140,28 @@ const outboundParsers = {
   toDeletePlaceholder: (line: DraftStockOutLine) => ({
     id: line.id,
   }),
-  toInsertServiceCharge: (line: DraftStockOutLine) => ({
+  toInsertServiceCharge: (
+    line: DraftStockOutLine
+  ): InsertOutboundShipmentServiceLineInput => ({
```

Comment:
added return types to hopefully reduce any naming changes being missed in future

Reply:
I really like de-structure and re-structure for these sorts of things, not sure if it would have helped here, but might have !